frontend/src/my-books/myBookModal/ShelfModal.css

```diff
@@ -0,0 +1,77 @@
+.shelf-modal-container {
+    display:flex;
+    flex-direction:column;
+    text-align: left;
+    padding-top: 32px;
+    padding-bottom: 32px;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  }
+  
+  .modal-title {
+    color:black;
+    font-size: 20pt;
```

Comment:
This is looking really great! Let's bump up the font-size for the .modal-title to 40pt. Double-check line spacing here as well and indent for new line on 46 and 57.

Reply:
Alrighty, now that I see the 40px, it is a bit too large. Let's take it down closer to where you had it, at 22px. We should be good then.